src/RG.h

```diff
@@ -38,3 +38,10 @@
 
 #endif
 
+
+// The unused macro should be applied to avoid compiler warnings
+// on set but unused variables
+
```

Comment:
Include `assert.h` in this file (compiler warning).

Reply:
👍